types/genesis_test.go

```diff
@@ -74,10 +74,12 @@ func TestBasicGenesisDoc(t *testing.T) {
 				"synchrony":  {"precision": "1", "message_delay": "10"},
 				"validator": {"pub_key_types":["ed25519"]},
 				"block": {"max_bytes": "100"},
-				"evidence": {"max_age_num_blocks": "100", "max_age_duration": "10"}
+				"evidence": {"max_age_num_blocks": "100", "max_age_duration": "10"},
+				"feature": {"vote_extension_enable_height": "0", "pbts_enable_height": "0"}
```

Comment:
Same here. Shouldn't we default to PBTS at this point?

Reply:
Good the same approach as for VE, not on by default.
We should probably have test cases for both situations from Genesis, though

Reply:
Will address in [#2324](https://github.com/cometbft/cometbft/issues/2324)